x-pack/plugins/observability_solution/synthetics/server/routes/monitor_cruds/services/delete_monitor_bulk.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { SavedObjectsClientContract, KibanaRequest } from '@kbn/core/server';
+import { SavedObject } from '@kbn/core-saved-objects-server';
+import { DEFAULT_SPACE_ID } from '@kbn/spaces-plugin/common';
+import { SyntheticsServerSetup } from '../../../types';
+import {
+  formatTelemetryDeleteEvent,
+  sendTelemetryEvents,
+} from '../../telemetry/monitor_upgrade_sender';
+import {
+  ConfigKey,
+  MonitorFields,
+  SyntheticsMonitor,
+  EncryptedSyntheticsMonitorAttributes,
+  SyntheticsMonitorWithId,
+} from '../../../../common/runtime_types';
+import { SyntheticsMonitorClient } from '../../../synthetics_service/synthetics_monitor/synthetics_monitor_client';
+import { syntheticsMonitorType } from '../../../../common/types/saved_objects';
+
+export const deleteMonitorBulk = async ({
```

Comment:
There's also a delete monitor bulk function on the file above. What's the difference here?